VMs/metrics/ubuntu16.04/env.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Copyright (c) 2018 Intel Corporation
+# 
+# SPDX-License-Identifier: Apache-2.0
+
+PRIVKEYNAME=$(pwd)/virsh.rsa
+PUBKEYNAME=${PRIVKEYNAME}.pub
+
+VM_NAME=ubnt16.04_master
```

Comment:
Is this a typo ("ubuntu"), or just truncated to fit into some limit?

Reply:
I think this name trimming is related to the libvirt XML related name rules https://libvirt.org/formatdomain.html#elementsMetadata
I'll see if I can come up with a fuller name that fits both the libvirt name and the subsequent uname, captures the distro better and allows us to distinguish between Master and Clone images...